packages/amplify-e2e-tests/src/__tests__/rds-v2.test.ts

```diff
@@ -147,4 +148,70 @@ describe("RDS Tests", () => {
     // PrimaryKey directive must be defined on Id field.
     expect(contactsIdFieldType.directives.find(d => d.name.value === 'primaryKey')).toBeDefined();
   });
+
+  // This test must be updated if the rds layer mapping file is updated
+  test("check the rds layer mapping file on the service account is available", async () => {
```

Comment:
I'm not sure about this test, this seems like it introduces fragility to the system, in terms of having tests fail randomly (from 1 developer's perspective) because a new layer is pushed.

I'm not firmly against it, but not sure it adds significant value. It does help us know that our layer publish mechanism works, but this seems like the wrong place to be confirming that.

Reply:
Currently publishing the S3 file with the latest layer config is a manual process. Since we do not have the integration tests yet, we don't want to say that the latest layer is safe to use.

The target is eventually to get to a state where all this is automated, we will publish the S3 file through our pipeline and we can add checking the availability of the S3 file to a canary. For now, this test verifies the availability of the file until we automate this.